File: Program's_Contributed_By_Contributors/Python_Programs/prime_number.py
def prime_number(n):
    c = 0
    for x in range(2, n):
        if n % x == 0:
            c = c + 1
    return c


n = int(input("Enter a number = "))
if prime_number(n) == 0:
    print("Prime number.")
else:
    print("Not prime number.")

File: Program's_Contributed_By_Contributors/c language/prime_number.c
#include<stdio.h>
int main()
{	int i,n,c=0;
	scanf("%d",&n);
	for(i=1;i<=n;i++)
	{if(n%i==0)
		c++;	}
	if(c==2){
	printf("it is prime number");
	}else{
		printf("not prime");
	}
}
